authentication/managing_cloud_provider_credentials/cco-short-term-creds.adoc

```diff
@@ -10,7 +10,7 @@ During installation, you can configure the Cloud Credential Operator (CCO) to op
 
 [NOTE]
 ====
-This credentials strategy is supported for Amazon Web Services (AWS), Google Cloud Platform (GCP), and global Microsoft Azure only. The strategy must be configured during installation of a new {product-title} cluster. You cannot configure an existing cluster that uses a different credentials strategy to use this feature.
+This credentials strategy is supported for {aws-first}, {gcp-first}, and global Microsoft Azure only. The strategy must be configured during installation of a new {product-title} cluster. You cannot configure an existing cluster that uses a different credentials strategy to use this feature.
```

Comment:
```suggestion
This credentials strategy is supported for {aws-first}, {gcp-first}, and global {azure-full} only. The strategy must be configured during installation of a new {product-title} cluster. You cannot configure an existing cluster that uses a different credentials strategy to use this feature.
```

Reply:
Attributes for full and first are the same, but wondering if first should be updated to Microshift Azure (Azure) in the attributes file?

Reply:
Yeah I have pondered this as well, but we don't use "Microshift Azure (Azure)" anywhere afaik so I have left it as full :)